python/doc/examples/reliability_sensitivity/estimate_probability_randomized_qmc.ipynb

```diff
@@ -40,9 +50,7 @@
   {
```

Comment:
The following line should be rephrased:
"# create the event we want to estimate the probability"
It should be:
"# create the event whose probability we want to estimate"

Reply:
Fixed.